packages/pixeloven/cli-addon-storybook/src/commands/story.test.ts

```diff
@@ -1,22 +1,109 @@
             });
             it("should contains required props", () => {
                 expect(storyModule.name).toEqual("story");
                 expect(typeof storyModule.run).toEqual("function");
             });
+            it("should exit with an error when no task is provided", async () => {
+                const context = await cli.run("story");
+                expect(context.commandName).toEqual("story");
+                expect(Stub.print.error.called).toEqual(true);
+                expect(
+                    Stub.print.error.calledWithExactly(
+                        "Invalid argument no task provided.",
+                    ),
+                ).toEqual(true);
+                expect(Stub.process.exit.called).toEqual(true);
+                expect(Stub.process.exit.calledWithExactly(1)).toEqual(true);
+            });
+            it("should exit with an error when an invalid task is provided", async () => {
+                const task = "wrong";
+                const context = await cli.run(`story ${task}`);
+                expect(context.commandName).toEqual("story");
+                expect(Stub.print.error.called).toEqual(true);
+                expect(
+                    Stub.print.error.calledWithExactly(
+                        `Invalid argument ${task}`,
+                    ),
+                ).toEqual(true);
+                expect(Stub.process.exit.called).toEqual(true);
+                expect(Stub.process.exit.calledWithExactly(1)).toEqual(true);
+            });
+            it("should exit with an error when a single invalid option is provided", async () => {
+                const task = "build";
+                const context = await cli.run(`story ${task} --wrong`);
+                expect(context.commandName).toEqual("story");
+                expect(Stub.print.error.called).toEqual(true);
+                expect(
+                    Stub.print.error.calledWithExactly(
+                        `Invalid ${task} option --wrong`,
+                    ),
+                ).toEqual(true);
+                expect(Stub.process.exit.called).toEqual(true);
+                expect(Stub.process.exit.calledWithExactly(1)).toEqual(true);
+            });
+            it("should exit with an error when multiple invalid options are provided", async () => {
+                const task = "build";
+                const context = await cli.run(`story ${task} --wrong --bad`);
+                expect(context.commandName).toEqual("story");
+                expect(Stub.print.error.called).toEqual(true);
+                expect(
+                    Stub.print.error.calledWithExactly(
+                        `Invalid ${task} options --wrong --bad`,
+                    ),
+                ).toEqual(true);
+                expect(Stub.process.exit.called).toEqual(true);
+                expect(Stub.process.exit.calledWithExactly(1)).toEqual(true);
+            });
+            // it("should complete build", async () => {
```

Comment:
Can't get these to mock/stub the storybook module. Tried several different methods and non of them worked. Figured I'd be better off trying to get this in and tested manually and then coming back to this.